open-models/serving/cloud_run_ollama_gemma2_rag_qa.ipynb

```diff
@@ -0,0 +1,1007 @@
+{
```

Comment:
<p>Would be good to stay consistent: "Gemma 2" rather than "Gemma2"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/968/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Reply:
Done thanks!